Production/python/Production.py

```diff
@@ -85,13 +85,21 @@
     process.combinatoricRecoTaus.builders[0].signalConeSize = cms.string('max(min(0.2, 4.528/(pt()^0.8982)), 0.03)') ## change to quantile 0.95
     process.selectedPatTaus.cut = cms.string('pt > 18.')   ## remove DMFinding filter (was pt > 18. && tauID(\'decayModeFindingNewDMs\')> 0.5)
 
-tauIdConfig = importlib.import_module('TauMLTools.Production.runTauIdMVA')
+tauIdConfig = importlib.import_module('RecoTauTag.RecoTau.tools.runTauIdMVA')
```

Comment:
I've switched to the `runTauIdMVA` code from official CMSSW, since the one used within TauMLTools does not work with Phase2 currently. Is there any particular reason to stick the TauMLTools specific version of it (apart from better handling of debug & input tau collection)?

Reply:
@lguzzi created a copy of the official `runTauIdMVA` to be able to re-run DeepTau after re-running tau reconstruction. I suggest to put all tau-id-related lines here inside `if else` for phase 2/run 2. This would also solve the two issues commented below.

Reply:
Done